src/core/tree_graphviz.ml

```diff
@@ -10,10 +10,11 @@
     see
     {{:https://graphviz.gitlab.io/_pages/doc/info/output.html}graphviz doc}.
 
-    To create dot files, use the [--write-trees] flag on command line: given a
-    module [M], a symbol [s] of the signature of [M], a file [M.s.gv]
-    containing the decision tree will be created. If [M] is [a.b.c], the file
-    will be [a/b/c.s.gv]. *)
+    To obtain the dot representation of a decision tree, use the
+    [decision-tree] command with a fully qualified symbol name to print the
+    graphviz code on standard output. To display symbol [sym] of module
+    [mod.ule] using [imagemagick],
+    [lambdapi decision-tree mod.ule.sym | dot -Tpng | display]. *)
```

Comment:
I think this can be removed completely, provided you put the information in the doc of the command (see my comment there).